test/framework/operator.go

```diff
@@ -216,3 +220,23 @@ func CreateNamespacedSubscriptionIfNotExist(namespace string, subscriptionName s
 func getOperatorImageNameAndTag() string {
 	return fmt.Sprintf("%s:%s", GetConfigOperatorImageName(), GetConfigOperatorImageTag())
 }
+
+// IsCommunityOperatorCrdAvailable returns whether the crd is available on cluster
+func IsCommunityOperatorCrdAvailable(operatorName string) (bool, error) {
+	operator, exists := KogitoOperatorCommunityDependencies[operatorName]
+	if !exists {
+		return false, fmt.Errorf("Unknown community operator %s", operatorName)
+	}
+	return IsCrdAvailable(operator.crdName)
+
+}
+
+// WaitForKogitoOperatorCrdAvailable waits for dependent operator main crd to be available
+func WaitForKogitoOperatorCrdAvailable(namespace, dependentOperator string) error {
+	if operatorInfo, exists := KogitoOperatorCommunityDependencies[dependentOperator]; exists {
+		return WaitFor(namespace, fmt.Sprintf("%s operator crd is available", dependentOperator), time.Minute*time.Duration(operatorInfo.timeoutInMin), func() (bool, error) {
```

Comment:
Do we want to incorporate the load factor also here?

Reply:
will be when kogito-1338 is merged into master ;)